test/common/index.js

```diff
@@ -355,7 +355,7 @@ let knownGlobals = [
   process,
   setImmediate,
   setInterval,
-  setTimeout
+  setTimeout,
```

Comment:
btw, this change is somewhat unrelated ;)

Reply:
Yes, I thought for a second I could add it here, but it's not the right place.
Reverted the comma.